src/webpage.cpp

```diff
@@ -1057,18 +1058,23 @@ QString WebPage::renderBase64(const QByteArray& format)
     return "";
 }
 
-QImage WebPage::renderImage()
+QImage WebPage::renderImage(bool onlyViewport)
```

Comment:
Hi Antonin (@astenfanutti), @ariya, can you please let me know if this issue  has this been fixed in phantomjs 2.1?  Or has the release changed to 2.2 (i hope it hasnt') for this issue? 
Thanks!